releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -2,6 +2,72 @@
 
 [Source code][source_code] | [Package (Maven)][package] | [Product documentation][docs] | [Samples][samples]
 
+## 1.1.1/1.2.1 - January 13, 2020
+
+* Added Support for Spring Boot 2.2.x Hoxton with version 1.2.1. Spring Boot 2.1.x Greenwich is still supported with 1.1.1.
+
+### Configuration
+
+* The property below has been removed as the timer-based configuration watch has been deprecated.
+
+```properties
+spring.cloud.azure.appconfiguration.watch.enabled
+```
+
+* The configuration is refreshed based on application activities.
+  * The configuration is cached and the default cache expiration time is 30 seconds. The configuration won't be refreshed until the cache expiration time window is reached. This value can be modified, see example below.
+  * In a web application, this library will signal configuration refresh automatically as long as there are incoming requests to the application. This is done by listening for ServletRequestHandledEvents.
+  * In a non-web application, the user needs to call AzureCloudConfigRefresh's `refreshConfigurations` method to signal configuration refresh at places where application activities occur. AzureCloudConfigRefresh can be accessed via dependency injection. Web based applications can also use this method, though both methods will still only update when the cache has expired. `refreshConfigurations` will return a `Future<Boolean>`, the resulting Boolean can be checked to see if a refresh event was triggered. **Note:** This does not mean the updated has been completed, only that a request was made to have the configurations updated.
+
+```properties
+spring.cloud.azure.appconfiguration.cache-expiration = 60
+```
+
+* Bug fix, configuration refresh occurred multiple times unnecessarily when an application loads configuration from more than one App Configuration store.
+* Bug fix, failed configuration refresh may not be reattempted.
+
+### Authentication
+
+* The property below has been removed as the previous AAD authentication has been replaced with the Azure Java SDK.
+
+```properties
+spring.cloud.azure.appconfiguration.managed-identity.object-id
+```
+
+* For clarity the name configuration has been changed to endpoint. Instead of my-configstore-name use `https://my-configstore-name.azconfig.io`
+
+```properties
+spring.cloud.azure.appconfiguration.stores[0].endpoint= https://my-configstore-name.azconfig.io
+```
+
+* A new authentication method has been added allowing users to provide there own credentials. Users can create a AppConfigCredentialProvider and KeyVaultCredentialProvider @Bean to provide credentials to App Configuration for connection to Azure App Configuration and Azure Key Vault respectively. Both objects define a method that is given a uri value and returns a [TokenCredential][token_credentials].
```

Comment:
* Users can now implement `AppConfigCredentialProvider` or `KeyVaultCredentialProvider` to use any of the `TokenCredential` types that is supported by the [Azure Identity](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/identity/azure-identity/README.md) to authenticate with App Configuration or Key Vault respectively. Please see xxx document for details.